atlasaction/action.go

```diff
@@ -568,7 +568,7 @@ func (a *Actions) MigrateAutoRebase(ctx context.Context) error {
 	if dirpath == "" {
 		dirpath = "migrations"
 	}
-	sumpath := filepath.Join(dirpath, "atlas.sum")
+	sumpath := filepath.Join(a.WorkingDir(), dirpath, "atlas.sum")
```

Comment:
when `working-directory` input was set (for example to `migrations`), the next command was failing:
```
git show origin/rebase:atlas.sum
```
since the command that should run is: `git show origin/rebase:migrations/atlas.sum` (from git project root)